src/components/CityColumnChart/CityColumnChart.js

```diff
@@ -107,20 +120,22 @@ const CityBarChart = (props) => {
     xAxis: {
       type: 'category',
       categories: data.map(city => city.name),
+      labels: xAxisLabelsConfig,
     },
     yAxis: {
       ceiling: props.ceiling,
       labels: {
+        padding: 0,
+        x: 0,
+        y: 0,
```

Comment:
`y: 3,` aligns better. But that's odd, because 3 should be the default. Or was zero on purpose so that the number sat above the line?